styles/Home.module.css

```diff
@@ -0,0 +1,122 @@
+.box{
+  width: fit-content;
+  height: fit-content;
+
+}
+.container {
+  min-height:1;
+  padding: 1 0 0 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  box-shadow: 1px;
+}
+.main {
+  padding: 1rem 0;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
+.box footer{
+  bottom: 0;
+}
+.title a {
+  color: #0070f3;
+  text-decoration: none;
+}
+.title a:hover,
+.title a:focus,
+.title a:active {
+  text-decoration: underline;
+}
+.title {
+  margin: 0;
+  line-height: 1.15;
+  font-size: 1.5rem;
+}
+.title,
+.description {
+  text-align: center;
+}
+.description {
+  line-height: 1.5px;
+  color: rgb(184, 13, 76);
+  font-size: 1.5rem;
+}
+.grid {
+  display: flex;
+  flex-wrap: wrap;
+  width: 80%;
+  margin-top: 1rem;
+}
+.card {
+  margin: 1rem auto;
+  padding: 0.5rem;
+  text-align: left;
+  color: inherit;
+  text-decoration: none;
+  border: 1px solid black;
+  border-radius: 10px;
+  transition: color 0.15s ease, border-color 0.15s ease;
+  width: 60%;
+}
+.card h2 {
+  margin: 0 0 1rem 0;
+  font-size: 1.5rem;
+  text-align: center;
+}
+
+.card p {
+  margin: 0;
+  font-size: 1.25rem;
+  line-height: 1.5;
+}
+.cardActions {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 7px;
+}
+.form {
+  width: 50%;
+  margin: 1rem auto;
+  padding: 10px;
+  box-shadow: 0 1px 3px 0 #d4d4d5, 0 0 0 1px #d4d4d5;
+}
+.formGroup {
+  width: 100%;
+  margin: 1rem 0px;
+}
+.formGroup label {
+  display: block;
+  margin-bottom: 0.5rem;
+}
+.formGroup input[type="text"] {
+  width: 100%;
+  padding: 10px;
+}
+.formGroup textarea {
+  width: 100%;
+  padding: 10px;
+}
+.error {
+  color: red;
+  text-align: center;
+}
+.success {
+  color: green;
+  text-align: center;
+}
+.success a {
+  color: blue;
+  text-decoration: underline;
+}
+@media (max-width: 600px) {
+  .grid {
+   width: 100%;
+   flex-direction: column;
+  }
+}
```

Comment:
Don't use these styles. Please stick to MUI's sx or styles components (from MUI)